packages/components/src/components/tooltip/tooltip.lite.tsx

```diff
@@ -0,0 +1,53 @@
+import { onMount, Show, useMetadata, useStore } from '@builder.io/mitosis';
+import { DBTooltipProps, DBTooltipState } from './model';
+import { cls, uuid } from '../../utils';
+import { DEFAULT_ID } from '../../shared/constants';
+
+useMetadata({
+	isAttachedToShadowDom: true,
+	component: {
+		// MS Power Apps
+		includeIcon: false,
+		properties: []
+	}
+});
+
+export default function DBTooltip(props: DBTooltipProps) {
+	// This is used as forwardRef
+	let component: any;
+	// jscpd:ignore-start
+	const state = useStore<DBTooltipState>({
+		_id: DEFAULT_ID,
+		handleClick: (event: any) => {
+			event.stopPropagation();
+		}
+	});
+
+	onMount(() => {
+		state._id = props.id || 'tooltip-' + uuid();
+		if (props.stylePath) {
+			state.stylePath = props.stylePath;
+		}
+	});
+	// jscpd:ignore-end
+
+	// TODO: Shall we check if only <span>, <p> or direct text was passed as children?
+	return (
+		<div
```

Comment:
```suggestion
		<span
```


`div` isn't possible to get used at all places / contexts, so we should switch to an inline-element like e.g. `span`.